src/frontend-global.scss

```diff
@@ -31,3 +31,6 @@
 
 /* Endowment */
 @import "styles/endowment-overrides";
+
+/* Report 2022-2023 */
+@import "styles/report-2022-2023";
```

Comment:
@sambulance Any opinions on how we might be able to avoid loading this CSS for other reports? Could we make this a "2022-2023 overrides" block, add that to last year's page, and enqueue these styles only if that block's present -- etc?

Reply:
Yes, I do want to load this conditionally. Does it need the complexity and overhead of a block though? Maybe we can do this based on page slug?

Reply:
I looked into this a little more, and discovered two things:

1. It's not so easy to just build a separate CSS file with `wp-scripts`, so we'd want to leverage blocks to build a separate asset. However:
2. All the styles for the custom blocks are added to all pages, regardless of the usage. 

This is the Research page, which is a standard content page unrelated to Reports, and the report block styles are being added.

<img width="1512" alt="Screenshot 2025-03-28 at 15 25 59" src="https://github.com/user-attachments/assets/2176c4f3-7e88-440f-a28e-3a5b8ea9c487" />

I'm going to merge this as is, and come back to the wider issue of loading stylesheets if we have time.